### ISSUE TITLE
How to set base_depth and base_dims on custom KITTI format dataset when using PGD?

### ISSUE THREAD

**Issue by clw5180** [open-mmlab/mmdetection3d#1748]
and what hyparamters needs to be modified? Thanks!

**Comment by ZCMax** [comment]
The `base_depth` and `base_dims` should be the average value of your training dataset corresponding ground truth labels
